content/docs/command-reference/unprotect.md

```diff
@@ -59,11 +59,11 @@ $ ls -lh
 -rw-r--r--  1 10576022 Nov 27 13:30 Posts.xml.zip
 
 $ dvc add Posts.xml.zip
-Saving information to 'Posts.xml.zip.dvc'.
+100% Add|██████████████████████████████████████████████|1/1 [00:00,  4.42file/s]
 
 To track the changes with git run:
 
-	git add .gitignore data.xml.dvc
+	git add data.xml.dvc .gitignore
```

Comment:
Thanks again. Same deal, let's please just remove all of these lines since the output isn't that relevant for the context.